app/javascript/tests/patient/Viewers.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Viewers from '../../components/patient/Viewers';
+import { mockPatient1, mockPatient2, mockPatient3 } from '../mocks/mockPatients';
+
+function getWrapper(mockPatient, viewers) {
+  return shallow(<Viewers patient_id={mockPatient.id} viewers={viewers} />);
+}
+
+describe('Viewers', () => {
+  const wrapper1 = getWrapper(mockPatient1, []);
+  const wrapper2 = getWrapper(mockPatient2, ["0@example.com"]);
+  const wrapper3 = getWrapper(mockPatient3, ["1@example.com", "2@example.com"]);
+
+  it('Properly renders a list of multiple viewers', () => {
+    expect(wrapper3.find('#multiple-warning').text().includes('1@example.com')).toBeTruthy();
+    expect(wrapper3.find('#multiple-warning').text().includes('2@example.com')).toBeTruthy();
+    expect(wrapper3.find('#multiple-warning').text().includes('Multiple users currently have this monitoree')).toBeTruthy();
+  });
+
+  it('Properly renders a list of a single viewer', () => {
+    expect(wrapper2.find('#multiple-warning').text().includes('0@example.com')).toBeTruthy();
+    expect(wrapper2.find('#multiple-warning').text().includes('Another user currently has this monitoree')).toBeTruthy();
+  });
+
+  it('Properly renders a list of an empty list of viewers', () => {
+    expect(wrapper1.hasClass('multiple-warning')).toBeFalsy()
```

Comment:
I'm not sure how that's really doing anything beyond what this check is already doing. And then I don't have to add the extra import.

Reply:
This is how we do it across the board, because it also tests that the import/component/etc is not there.  We do it for child components, form inputs, etc, so I'd prefer it to be done that way for consistency

Reply:
That seems fair enough, made this change in the latest squashed commit.

Reply:
I'm going to be super pick and say that you should check for `Alert` first and then the text, but otherwise the front end parts look good to me!

Reply:
you got it: b7b0b61ee22cb51cc6c93bf879135e64894bdf83